todo_app/app.py

```diff
@@ -8,4 +9,21 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    all_items = get_items()
+    todos, completed = [], []
+    for item in all_items:
+        (completed if item['status'] == 'Completed' else todos).append(item)
```

Comment:
This is a really neat way to do it, but might be less clear than using a list comprehension directly:
```suggestion
    todos = [item for item in all_items if item['status'] != 'Completed']
    completed = [item for item in all_items if item['status'] == 'Completed']
```

Reply:
vg.
all this is removed by now anyway for the next exercise, but thoughts taken on. ta